src/WallGo/mathematicaHelpers.py

```diff
@@ -1,22 +1,70 @@
-import subprocess
 import pathlib
+import subprocess
+import sys
+
+# Put common Wolfram Mathematica and WallGoMatrix related functions here.
+# Common physics/math functions should go into helpers.py
+
+
+def generateMatrixElementsViaSubprocess(
+    inFilePath: pathlib.Path, outFilePath: pathlib.Path, bVerbose: bool = False
+) -> None:
+    """
+    Generates matrix elements by executing a Mathematica script via a subprocess.
+
+    This function takes the input and output file paths, converts them to string representations,
+    and constructs a command to run a Mathematica script using `wolframscript`. The command is
+    executed using the `subprocess.run` method, and the output is printed to the console. If the
+    command fails, an error message is printed.
+
+    This requires an active WolframKernel.
+
+    Args:
+        inFilePath (pathlib.Path): The path to the input file containing the Mathematica script.
+        outFilePath (pathlib.Path): The path to the output file where the results will be saved.
+
+    Raises:
+        subprocess.CalledProcessError: If the subprocess command fails.
+    """
+    # Ensure filePath is string representation of the path
+
+    inFilePathStr = str(inFilePath)
+    outFilePathStr = str(outFilePath)
+
+    banner = f"""\n
+================================================
+    WallGoMatrix: Recomputing Matrix Elements
+================================================
 
-## Put common matheamtica and DRalgo related functions here. Common physics/math functions should go into helpers.py
+    Matrix elements are being recomputed from
+    Input file  : {inFilePathStr}
 
+    Matrix elements are being stored at 
+    Output path: {outFilePathStr}
+    using WallGoMatrix.
+    
+    Please wait while the computation is in progress...
+================================================
+"""
+    print(banner)
```

Comment:
Not a fan of having a long print like this. How about just one line: "Generating matrix elements from {inFilePath} to {outFilePath}" ?

Reply:
Needed to make this distinct from the WallGo python output. But I can make the printout smaller.